Simple Inventory Management System/Simple Inventory Management System/Inventory .cs

```
@@ -0,0 +1,126 @@
+﻿
+namespace Simple_Inventory_Management_System
+{
+    public class Inventory
+    {
+        public List<Product> Products { get; private set; }
+        public Inventory(List<Product> products)
+        {
+            this.Products = products;
+        }
+        public Inventory()
+        {
+            this.Products = new List<Product>();
+        }
+        public bool Add(Product product)
+        {
+            if (IsNotValid(product))
```

if the product is not valid, then we should throw an exception indicating that.